src/routes/safe/components/Apps/ListContentLayout/List.jsx

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import React from 'react'
+import styled from 'styled-components'
+import cn from 'classnames'
+import { withStyles } from '@material-ui/core/styles'
+
+// TODO: move these styles to a generic place
+import { styles } from '../../Settings/style'
+
+const Wrapper = styled.div``
+
+const Item = styled.div`
+  border-bottom: solid 2px rgb(232, 231, 230);
+
+  .container {
+    display: flex;
+    align-items: flex-end;
+  }
+`
+
+const IconImg = styled.img`
+  width: 20px;
+  margin-right: 10px;
+`
+
+function List({ items, activeItem, onItemClick, classes }: any) {
```

Comment:
we use arrow functions across the project so please change this

Reply:
done